### PR TITLE
Temoa: Expands transportation sector, specific nuclear turbine

### LINKED ISSUE #43: Add Transportation Sector for Temoa Model
This issue can be closed when 
- [ ] a ``.sql`` database has been added to the repository that includes a transportation sector for UIUC
- [ ] using data about gasoline/diesel demand. All transportation demand should be in a "gasoline energy equivalent."

The file should be called ``0x_uiuc.sql`` and have an associated config file called ``run_scenario0x.txt``, similar to the other scenarios in ``temoa-uiuc``.

### Transportation Demand
All of the data analysis for transportation demand has been done in ``fuel-analysis``, ``hydro-requirement``, and ``uiuc_transportation_demand``. This should be enough information to update the ``TEMOA`` model.

### Guidance:
1. Add an experimental branch in your forked repository

``git checkout -b iss43``

2. Each subsequent ``TEMOA`` model should build on the previous one (in most cases). Thus, copy the most recent model, in this case ``scenario04``. 
```bash
cp data_files/04_uiuc.sql data_files/05_uiuc.sql
cp data_files/run_scenario04.txt data_files/run_scenario04.txt
```

3. Edit the configuration file, ``run_scenario05.txt`` and change the name of the input database to ``data_files/05_uiuc.sqlite``
   * Also change the scenario name, and the output file name.

4. In ``Snakefile`` comment out the "All Scenarios" block, and uncomment the "Experimental Scenario" block. Replace the name in experimental scenario with the name of the model you're working on.

5. Edit the ``.sql`` database to add a new sector, transportation. (You can visually check the model at model.temoacloud.com)

6. Run the model by executing
```bash
source activate temoa-py3
snakemake --cores=4
```
 
7. If the results look appropriate, add the experimental scenario to all scenarios in the snakefile (maybe run it again with all files) and make a pull request.

### PR DESCRIPTION
**Pull request by samgdotson**
This PR adds two iterations on the temoa UIUC model:
1. Diesel and E85
2. Specific turbine for the nuclear reactor
  - Abbott is a combined heat and power plant which has a much higher thermal efficiency than a nuclear reactor.

Closes #43

### CODE REVIEW

File: temoa-uiuc/Snakefile [hunk 1]
```diff
@@ -9,9 +9,9 @@
 #           "05_uiuc_scenario05_model"]
 
 # Experimental Scenario
-DBS = ["06_uiuc"]
-SCENARIOS = ["scenario06"]
-FOLDER = ["06_uiuc_scenario06_model"]
+DBS = ["07_uiuc"]
+SCENARIOS = ["scenario07"]
+FOLDER = ["07_uiuc_scenario07_model"]
```
Comment (yardasol) [inline temoa-uiuc/Snakefile:14]: okay

File: temoa-uiuc/data_files/07_uiuc.sql [hunk 2]
```diff
@@ -0,0 +1,1224 @@
+/*
+This scenario adds a transportation sector to UIUC.
+All transportation demand is measured by the energy equivalent of gasoline.
+*/
```
Comment (yardasol) [inline temoa-uiuc/data_files/07_uiuc.sql:4]: Good docs :)

File: temoa-uiuc/data_parser.py [hunk 3]
```diff
@@ -25,7 +25,7 @@
 #              'emissions': 'V_EmissionActivityByPeriodAndProcess'}
 time_horizon = np.arange(2021, 2031, 1)
 
-elc_techs = ['IMPELC', 'IMPSOL', 'IMPWIND', 'TURBINE']
+elc_techs = ['IMPELC', 'IMPSOL', 'IMPWIND', 'TURBINE', 'NTURBINE']
```
Comment (yardasol) [inline temoa-uiuc/data_parser.py:28]: nuclear turbine!!!

File: temoa-uiuc/data_files/run_scenario07.txt [hunk 4]
```diff
@@ -0,0 +1,45 @@
+#-----------------------------------------------------
+# This is a sample configuration file for Temoa
```
Comment (yardasol) [inline temoa-uiuc/data_files/run_scenario07.txt:2]: I like the detail of the comments here! Very helpful.

**Review by yardasol — APPROVED** [review]
Overall looks good. I'll go ahead and make an issue for adding gudiing comments to .sql files to make them a bit easier to read (hopefully)

**Comment by robfairh** [comment]
Hitting the merge button

**Pull request merged**